src/LavaNode.cs

```diff
@@ -298,26 +299,26 @@ await voiceChannel.DisconnectAsync()
     /// </summary>
     /// <param name="trackHash"></param>
     /// <returns></returns>
-    public async Task<TLavaTrack> DecodeTrackAsync(string trackHash) {
+    public async Task<DecodedLavaTrack> DecodeTrackAsync(string trackHash) {
         ArgumentNullException.ThrowIfNull(trackHash);
-        var responseMessage = await _httpClient.GetAsync($"/{_version}/decodetrack?encodedTrack={trackHash}");
+        var responseMessage = await _httpClient.GetAsync($"/{_version}/decodetrack?encodedTrack={HttpUtility.UrlEncode(trackHash)}");
```

Comment:
Isn't trackHash already an encoded string?

Reply:
At least on both `TrackEndEventArg` and `TrackStartEventArg`, `encodedTrack` isn't URL-safe.